### DISCUSSION TITLE
Feature Request (Scroll on taksbar icon to increase & decrease volume)

### DISCUSSION THREAD

**Discussion by BiazPrasastha** [File-New-Project/EarTrumpet#883] [Ideas]
I'm using this kind of feature on 'Twinkle Tray' to control monitor brightness, maybe you can implement this on EarTrumpet in the next update

**Comment by davuxcom** [comment]
Scrolling using the mouse wheel on the EarTrumpet icon will already change the master volume. This has been in EarTrumpet for a year or more, including the last few versions.  Are you asking for something else or did you miss it or find it broken?

**Reply by BiazPrasastha** [reply]
okay thanks, but i the scroll feature doesn't work when i'm using my touchpad, it only work when using scroll wheel
